### DISCUSSION TITLE
Add Jellyseer to have a more complete media streaming server

### DISCUSSION THREAD

**Discussion by boterocamilo** [MichaIng/DietPi#7000] [Software Request]
#### Formal software information
- Software title | **Jellyseerr**
- Short description | (Owner description) It is a free and open-source software application for managing requests for your media library. It is a fork of Overseerr built to bring support for Jellyfin & Emby media servers.
- Official URL (if available) | None
- GitHub URL (if available) | https://github.com/Fallenbagel/jellyseerr
- (Official) install documentation | https://github.com/Fallenbagel/jellyseerr?tab=readme-ov-file#linux

#### Are there similar/alternative software titles available with DietPi-Software?

- There are not.

#### What makes your requested software better than the above solutions, if available?

- ...

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?

- ...

#### Can you provide the installation steps that you would suggest DietPi-Software to do?

1. ...

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.

- Unfortunately, I do not have the skills for it, despite I would like.

**Comment by NiklasWallin** [comment]
Any more info about adding Jellyseerr to the dietpi optimized software list?
Jellyseerr would pretty much "complete" the sonarr / radarr / transmission media server and management.
Thanks.

**Comment by paeore** [comment]
Be cool to at least get an answer as to whether this is even achievable as I feel this would be a great addition as others have mentioned.

Thanks as always for your amazing work!!

paeore
